lib/core/renderer.js

```diff
@@ -249,12 +249,12 @@ module.exports = class Renderer {
     }
 
     // For serving static/ files to /
-    this.useMiddleware(
-      serveStatic(
-        resolve(this.options.srcDir, 'static'),
-        this.options.render.static
-      )
+    const staticMiddleware = serveStatic(
+      resolve(this.options.srcDir, 'static'),
+      this.options.render.static
     )
+    staticMiddleware.prefix = this.options.render.static.prefix
```

Comment:
Code readability suggestion: `Boolean(this.options.render.static.prefix)`.

Reply:
It may make unexpected behavior, if not config `static.prefix`, it will be false and ignore the prefix, but prefix should not be ignored by default.

Reply:
So what if we add a default value `true` in https://github.com/nuxt/nuxt.js/blob/215832769edc42e7077e8db970fbcda3276ecf0a/lib/common/options.js#L308? This prefix is little confusing now.

Reply:
@pi0 Added the default value.